core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -605,6 +607,84 @@ public FileSystemMasterView getFileSystemMasterView() {
   }
 
   /**
+   * Checks the consistency of the subtree under the path.
+   *
+   * @param path the subtree root to check
+   * @param options the options to use for the checkConsistency command
+   * @return a list of paths in Alluxio which are not consistent with the under storage
+   * @throws FileDoesNotExistException if the path does not exist
+   * @throws InvalidPathException if the path is invalid
```

Comment:
I see your point - I would still prefer to remove the exception so that it's one less backcompat break for users to deal with, but if you value consistency higher I don't feel strongly about it.